frontend/src/components/admin/CoursePreview.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import ModulePreview from "./ModulePreview";
+import { CoursePreviewProps } from "../../types/AdminDashboardTypes";
+
+const CoursePreview = ({
+  title,
+  description,
+  isPrivate,
+  modules,
+}: CoursePreviewProps): React.ReactElement => {
+  return (
+    <div>
+      Course Preview
+      {title}
+      {description}
+      {isPrivate}
+      {modules.map((x, i) => (
```

Comment:
NBD since it's all prototype, but can we eventually give these better names?

Reply:
Yeah totally. Do you mean the props or the files?